docs/README.adoc

```diff
@@ -29,3 +29,19 @@ link:./security.adoc[security.adoc] - A brief description of properties a system
 link:./selectively-triggering-aktualizr.adoc[selectively-triggering-aktualizr.adoc] - Aktualizr normally runs as a service, polling for updates periodically. However, it can also be triggered externally--for example, in response to a shoulder-tap, or in response to user interaction on an HMI. This document describes how to do that.
 
 link:./integrating-libaktualizr.adoc[integrating-libaktualizr.adoc] - How to use Aktualizr as a library ('libaktualizr') to implement custom software update clients. This can be used to integrate Aktualizr with a user interface or to deliver updates to a custom package manager.
+
+== Developer documentation
+
+Additional documentation intended for developers that may need to use the libaktualizr API should refer to link:https://advancedtelematic.github.io/aktualizr/index.html[the doxygen output on github].
+
+=== Updating doxygen on github
+
+To update the doxygen documentation on github, you will need to do something like the following:
+
+1. In an aktualizr repo, run `make docs` in the build directory.
```

Comment:
Every item starts with `1. `. Is it intentional?

Reply:
Yep, when it's rendered, the numbering is automatic: https://github.com/advancedtelematic/aktualizr/blob/169013c20e14cfb22268b7509ca0676a9ac41b16/docs/README.adoc

Reply:
Ok thanks. Could have looked that up I guess, sorry.